packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -225,10 +236,13 @@ export async function getBookings({
         seatsShowAvailabilityCount: true,
         eventTypeColor: true,
         schedulingType: true,
+        length: true,
         team: {
           select: {
             id: true,
             name: true,
+            members: true,
+            slug: true,
```

Comment:
Do we need to fetch all the members of team? For large orgs and teams this doesn't load fast

Reply:
Looks like not. Thanks for catching !!